apps/gitness/src/framework/context/AppContext.tsx

```diff
@@ -47,26 +41,47 @@ export const AppProvider: React.FC<{ children: ReactNode }> = ({ children }) =>
   }, [])
 
   useEffect(() => {
-    if (token) {
+    if (isAuthorized) {
       membershipSpaces({
         queryParams: { page: 1, limit: 10, sort: 'identifier', order: 'asc' }
       }).then((response: MembershipSpacesOkResponse) => {
         if (response.length > 0) {
           const spaceList = response.filter(item => item?.space).map(item => item.space as TypesSpace)
           setSpaces(spaceList)
+        } else {
+          window.location.href = '/create-project'
         }
       })
```

Comment:
should we debounce the call like would it repeat calls?

Reply:
I removed it